### DISCUSSION TITLE
The current documentation for Apptainer is no longer in its own section, but ended up in the "Advanced settings" section of Singularity

### DISCUSSION THREAD

**Discussion by lckarssen** [nextflow-io/nextflow#4345] [Q&A]
## Bug report 

[The current "latest" documentation](https://www.nextflow.io/docs/latest/container.html#containers) doesn't have an Apptainer subsection in the "Containers" section, whereas [the current `edge` documentation](https://www.nextflow.io/docs/edge/container.html#apptainer) has. The same holds for the "Configuration" → "Configuration scopes" section. Searching through the `latest` HTML docs I found the pieces of text from the `edge` docs in the "Advanced settings" section below "Singularity ([here](https://www.nextflow.io/docs/latest/container.html#id30) and [here](https://www.nextflow.io/docs/latest/config.html#scope-singularity), respectively for the "Containers" section and the "Configuration scopes" section. 

So the documentation code is there, and, in fact, the code for [`docs/containers.rst`](https://github.com/nextflow-io/nextflow/blob/STABLE-23.04.x/docs/container.rst) and [`docs/config.rst`](https://github.com/nextflow-io/nextflow/blob/STABLE-23.04.x/docs/config.rst) hasn't changed [since the original commit](https://github.com/nextflow-io/nextflow/pull/3700/files) (#3700) by @matthdsm on 2023-03-01. 

Could this be some kind of difference in the way the documentation is generated between `latest` and `edge`?

**Comment by mribeirodantas** [comment]
Hi, @lckarssen.

Whenever we have a new stable release, the stable version of the documentation will be updated. Whenever we have a new edge release, the edge version of the documentation will be updated. This means that the bleeding edge version of the documentation can be found on GitHub, then edge, then latest, in this order.

With time, what you see in the edge version of the documentation will [probably] be seen in the latest version.

**Reply by lckarssen** [reply]
Hi @mribeirodantas ,

Tunnels for you reply. The flow you describe is how I thought it would be. However, that still leaves me confused. The documentation related to Apptainer hasn't changed since it was first merged in #3700 back in March. It has since been part of several 22.x edge releases, and then made it into 23 stable (23.04.0 IIRC). And indeed I do see the documentation in the latest version, it is just in the wrong place when rendered on the documentation site. Since the documentation code in the `.rst` hasn't changed, I am wondering how the text ends up in the wrong section in the HTML version on the "latest" documentation site, but is correctly rendered in the "edge" docs. 

As the `.rst` files haven't changed, does that mean the code that rendered the documentation has changed between "latest" and "edge"?

To be sure,  the "latest" documentation is for the latest stable release,  right? 

Is there a way to see for which version/ commit the rendered documentation is?

**Reply by mribeirodantas** [reply]
You're looking at the incorrect files. We've ported the documentation to Markdown, we're not using `rst` files anymore. You can see the Apptainer subsection in the container section [here](https://github.com/nextflow-io/nextflow/blob/master/docs/container.md).
